### PR TITLE
Removes shadowlings

### PR DESCRIPTION
**Pull request by k-k-karma**
:cl: karma
del: Shadowlings are gone
/:cl:

### CODE REVIEW

**Comment by Kawaii-Big-Boss** [comment]
Do this via the config.
But +1.

**Comment by Lxtoofonea** [comment]
Yeah I'm gonna do that when I get back home, but still have a +1

**Comment by yskefunky** [comment]
no fuck off, don't delete them entirely make them still admin spawnable, just remove them off roundstart antag list. I need good memes every once in a while ok papi
-1

**Comment by yobob591** [comment]
Im gonna have to agree, theres no reason to remove the gamemode from the game, just remove it from rotation so -1

File: hippiestation/code/modules/mob/mob_movement.dm [hunk 1]
```diff
@@ -1,53 +1,6 @@
 #define MAX_SW_LUMS 0.2
```
Comment (ghost) [inline hippiestation/code/modules/mob/mob_movement.dm]: these 2 defines can go too

**Comment by bardofrage** [comment]
I'm agreeing with yobob here, just remove it from rotation. -1

**Pull request closed without merging**